### DISCUSSION TITLE
Why cell types with much more cell numbers have less union peaks?

### DISCUSSION THREAD

**Discussion by YayanFeng55** [GreenleafLab/ArchR#1979] [Questions / Documentation]
Dear all,

I am working on a multiome dataset. I have a question about 'addReproduciblePeakSet' function. I have annotated a large cell identify, which account for approx. 72% of the total nucleus. Just as below:

celltypeA: nucleus n = 78458, union peak number n = 44621
celltypeB: nucleus n = 3421, union peak number n = 72515
celltypeC: nucleus n = 5337, union peak number n = 51980

In 'addReproduciblePeakSet' function, I used:

pathToMacs2 <- findMacs2()
projMulti3 <- addReproduciblePeakSet(
     ArchRProj = projMulti3,
	 groupBy = "Clusters_Combined",
	 minCells = 25,
	 pathToMacs2 = pathToMacs2
)

Why cell types with much more cell numbers have less union peaks? How can I set addReproduciblePeakSet to obtain a reasonable union peak number ?

Thanks
Yayan

**Comment by rcorces** [comment]
This is not a specific enough question to provide a specific answer. I also think you are misinterpreting something about the union peak set, Specifically this - https://www.archrproject.com/bookdown/calling-peaks-w-macs2.html#:~:text=This%20peak%20set%20contains%20an%20annotation%20for%20the%20group%20from%20which%20each%20peak%20originated.%20However%2C%20these%20annotations%20do%20not%20inherently%20mean%20that%20the%20given%20peak%20was%20only%20called%20in%20that%20group%2C%20rather%20that%20the%20annotated%20group%20had%20the%20highest%20normalized%20significance%20for%20that%20peak%20call.

**Comment by YayanFeng55** [comment]
Hi Ryan,

Thank you so much for your explanation, you really helped!

Thanks
Yayan
